packages/status-page/src/utils/getNumProvers.ts

```diff
@@ -14,5 +14,7 @@ export const getNumProvers = async (
   const resp = await axios.get<UniqueProverResponse>(
     `${eventIndexerApiUrl}/uniqueProvers`
   );
-  return resp.data;
+  let uniqueProverRes = resp.data;
+  uniqueProverRes.provers.sort((a, b) => b.count - a.count);
+  return uniqueProverRes;
```

Comment:
NITPICK:

```js
  const uniqueProversResp = await axios.get<UniqueProverResponse>(
    `${eventIndexerApiUrl}/uniqueProvers`
  );
  return uniqueProverResp.data.provers.sort((a, b) => b.count - a.count);
```

Reply:
Would be good to add safeguards for when the API doesn't return `data.provers` i.e `data` is empty.

Reply:
@wolfderechter can you change it to this:
```ts
 const uniqueProversResp = await axios.get<UniqueProverResponse>(
    `${eventIndexerApiUrl}/uniqueProvers`
  );
  return (uniqueProverResp.data?.provers || []).sort((a, b) => b.count - a.count);
  ```

Reply:
> @wolfderechter can you change it to this:

`Type 'UniqueProver[]' is missing the following properties from type 'UniqueProverResponse': uniqueProvers, proversts(2739)`
Getting this error, what do you think about this solution @cyberhorsey?

```ts
  const uniqueProversResp = await axios.get<UniqueProverResponse>(
    `${eventIndexerApiUrl}/uniqueProvers`
  );
  uniqueProversResp.data?.provers.sort((a, b) => b.count - a.count);
  
  return uniqueProversResp.data || { uniqueProvers: 0, provers: [] };
```